src/cfn.py

```diff
@@ -41,11 +44,31 @@ def ensure_destroyed(stackname):
 @task(alias='aws_update_stack')
 @requires_aws_stack
 @timeit
-def update(stackname):
+def update(stackname, part_filter=None):
```

Comment:
Proposals:
- `filter` is too generic and shadows the `filter` built-in function
- `resource_filter` is more precise but you have to understand it's referring to the template's resources. However you may have two EC2 nodes but it's the filter is still just `ec2`
- `service_filter` is how AWS would call it, `ec2` and `s3` are `Services` in their console, still `service` has many meanings in this world
- `aws_service_filter` starts to become too long
Choice?